mirage/serializers/registration.ts

```diff
@@ -114,6 +114,13 @@ export default class RegistrationSerializer extends ApplicationSerializer<Mirage
                     },
                 },
             },
+            resources: {
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/registrations/${model.id}/resources/`,
```

Comment:
Do we need a `meta` object here?

Reply:
The meta object wasn't called for by the API contract I don't believe (well, there was one, but it was empty).